src/features/header-flyout/styles/HeaderFlyout.scss

```diff
@@ -19,6 +20,14 @@
     }
 }
 
+@include breakpoint($medium-screen) {
+    .responsive-flyout {
```

Comment:
Do we need this specificity or can we target `.flyout-list-container-title` directly?